edb/server/ext/smtp.py

```diff
@@ -0,0 +1,174 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2023-present MagicStack Inc. and the EdgeDB authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import *
+
+import asyncio
+import email
+
+import aiosmtplib
+
+from edb.common import retryloop
+from edb.ir import statypes
+
+from . import util
+from ... import errors
```

Comment:
Please use absolute imports for when we're importing from parents and grandparents